org.librarysimplified.audiobook.views/src/main/java/org/librarysimplified/audiobook/views/PlayerSleepTimerFragment.kt

```diff
@@ -100,7 +82,7 @@ class PlayerSleepTimerFragment : DialogFragment() {
 
   private fun enabledSleepTimerConfigurations(): List<PlayerSleepTimerConfiguration> {
     val nowEnabled =
-      this.context!!.resources.getBoolean(R.bool.audiobook_player_debug_sleep_timer_now_enabled)
+      this.requireContext().resources.getBoolean(R.bool.audiobook_player_debug_sleep_timer_now_enabled)
```

Comment:
Glad that you are removing these forced unwrappings

Reply:
Yeah I'm not trying to rewrite these (yet) but nowadays Android Studio gives errors for force-unwrapping contexts, fragment arguments, etc.